CHANGELOG.md

```diff
@@ -13,6 +13,11 @@ This changelog documents all notable user-facing changes of VAST.
 
 ## Unreleased
 
+- 🐞 VAST no longer crashes when the disk monitor tries to calculate the size of
+  the database while files are being deleted. Instead, it will retry after the
+  configured scan interval.
+  [#1458](https://github.com/tenzir/vast/1458)
```

Comment:
The delta to the recent refactoring is almost so small that this doesn't warrant an entry. No strong opinion though. We can also keep it.

Reply:
Let's keep it in. It's a bug fix after all.